kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -299,4 +302,56 @@ int runInit() {
   boolean dispatch(String line) {
     return super.dispatch(isPythonMode() ? line : HiveStringUtils.removeComments(line));
   }
+
+  @Override
+  void handleSQLException(SQLException e) {
+    if (e instanceof SQLWarning && !(getOpts().getShowWarnings())) {
+      return;
+    }
+
+    if (e.getCause() instanceof TTransportException) {
+      switch (((TTransportException) e.getCause()).getType()) {
+        case TTransportException.ALREADY_OPEN:
+          error(loc("hs2-connection-already-open"));
+          break;
+        case TTransportException.END_OF_FILE:
+          error(loc("hs2-unexpected-end-of-file"));
+          break;
+        case TTransportException.NOT_OPEN:
+          error(loc("hs2-could-not-open-connection"));
+          break;
+        case TTransportException.TIMED_OUT:
+          error(loc("hs2-connection-timed-out"));
+          break;
+        case TTransportException.UNKNOWN:
+          error(loc("hs2-unknown-connection-problem"));
+          break;
+        default:
+          error(loc("hs2-unexpected-error"));
```

Comment:
If it is Hiveserver2 connected by kyuubi beeline, the exception returned by Hiveserver2 is `org.apache.thrift.transport.TTransportException`. 

It should not be matched here?

cc @zhouyifan279

Reply:
My description here is wrong, please ignore it.
Thank you for your contribution!
LGTM